src/Listener/AbstractListener.php

```diff
@@ -20,17 +20,12 @@ public function __construct(?ListenerOptions $options = null)
         $this->setOptions($options);
     }
 
-    /** @return ListenerOptions */
-    public function getOptions()
+    public function getOptions(): ListenerOptions
```

Comment:
These changes will break any existing child class implementation: we can't add native types without breaking inheritance.

For example, see https://3v4l.org/75MVt

Ref: https://github.com/laminas/laminas-ci-matrix-action/pull/126 (that patch saves me SO MUCH work in reviews!)